client/src/components/DashboardCoponent/Content.tsx

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import React, { useState } from 'react';
+import Card from './ProfileCard';
+import { Grid, Button, Box } from '@material-ui/core';
+import { makeStyles } from '@material-ui/styles';
+
+const useStyles = makeStyles(() => ({
+  bottomButton: {
+    margin: 'auto',
+  },
+  btnContainer: {
+    padding: '2rem 0',
+    display: 'grid',
+    width: '100%',
+  },
+}));
+
+function Content() {
+  const [length, setLength] = useState(6);
+
+  const classes = useStyles();
+
+  const toggelLengthHandler = () => {
```

Comment:
When you implement pagination, you can use MUI Pagination component.

Reference: https://material-ui.com/components/pagination/